Makefile

```diff
@@ -74,7 +74,7 @@ docker-build-nix: docker-nix-base
 docker-alpine-base: $(go_package_cache_dir)
 	@if ! $(docker) images|grep katzen/alpine_base; then \
 		$(docker) run --replace --name katzen_alpine_base docker.io/golang:alpine \
-		sh -c 'apk add bash gcc musl-dev libxkbcommon-dev pkgconf wayland-dev \
+		sh -c 'apk add bash git gcc musl-dev libxkbcommon-dev pkgconf wayland-dev \
```

Comment:
I see you added git to the alpine image here, was that intentional?

Reply:
I some environments proxy.golang.org blocks network connections so if GOPROXY=direct is set, git is needed.